.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Hmm could we simply build all branches?

```suggestion
on: [push, pull_request]
```

---

Then you build branches twice, once due to the push and once due to the PR change.

Unless you really need to trigger CI for branches without an open PR, that's not necessary.

---

> Then you build branches twice, once due to the push and once due to the PR change.

Sure, but who cares since CI on this project takes less than 1 minute (and usually both jobs would run in parallel anyway, so no additional time is needed) :wink:

> Unless you really need to trigger CI for branches without an open PR, that's not necessary.

I often want to run CI without opening a PR (yet). In addition, with the current configuration even tags are not checked on CI...

---

Ok, fixed.